src/components/core/error/ErrorUnsupportedNetwork.tsx

```diff
@@ -2,15 +2,18 @@ import { ReactComponent as IconWarning } from 'assets/icons/warning.svg';
 import { Button } from 'components/common/button';
 import { useWeb3 } from 'libs/web3';
 import { ErrorWrapper } from 'components/core/error/ErrorWrapper';
+import config from 'config';
 
 export const ErrorUnsupportedNetwork = () => {
   const { disconnect, switchNetwork } = useWeb3();
 
+  const networkName = config.network.name;
+
   return (
     <ErrorWrapper
       icon={<IconWarning />}
       title="Wrong Network"
-      text="Please connect to Ethereum Mainnet using your wallet or the button below"
+      text={`Please connect to the ${networkName} using your wallet or the button below`}
```

Comment:
The "the" should be removed. It will end up be "please connect to the ethereum mainnet"